javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -562,8 +562,10 @@ void PrintES6Imports(Printer* printer, const FileDescriptor* file) {
     }
     imports.insert(dep_filename);
     // We need to give each cross-file import an alias.
-    printer->Print("import * as $alias$ from '$dep_filename$_pb';\n", "alias",
-                   ModuleAlias(name), "dep_filename", dep_filename);
+    printer->Print("import * as $alias$ from '$dep_filename$_pb'; // proto import: \"$proto_import$\"\n",
+                   "alias", ModuleAlias(name),
+                   "dep_filename", dep_filename,
+                   "proto_import", name);
```

Comment:
nit: do you mind renaming `name` => `proto_filename`, if that's what it is?

Reply:
Done. However, I moved the PR to use my corporate github account for CLA purposes. https://github.com/grpc/grpc-web/pull/1313. I will close this PR. Please take a look at the new one.

Reply:
cool thanks will do!